lib/rucio/tests/test_automatix.py

```diff
@@ -81,3 +84,5 @@ def test_automatix(vo, root_account, rse_factory):
         assert status
         for file_ in files:
             assert file_dict["%s:%s" % (file_["scope"], file_["name"])] is True
+    config_remove_option("automatix", "rses")
```

Comment:
did you try using the `file_config_mock` fixture for this ? If the automatix test doesn't interact using the API with the rest server, it should be a better solution than marking the test noparallel.

Reply:
There are 2 issues here :
- Some options depends on variables (e.g. rse coming from rse_factory) that are generated by the test itself
- Other options are conditional, based on environment variable (POLICY) and I don't think that the `file_config_mock` allows to do it